test/components/Sokeresultat.test.tsx

```diff
@@ -37,7 +37,6 @@ describe('Sokeresultat', () => {
   it('Skal inneholde knapperad', () => {
     renderSokeresultat();
     expect(screen.getByRole('button', { name: 'Tildel veileder' })).to.exist;
-    expect(screen.getByRole('button', { name: 'Søk veileder (0)' })).to.exist;
```

Comment:
Evt om det kan lages en ny test for ny filtrering?

Reply:
Det hørtes ut som en god test! 🤌 Kan prøve meg på noe sånt